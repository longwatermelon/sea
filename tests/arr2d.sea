fn main() -> int {
    let arr:int** = stalloc(2, int*);
    arr[0] = stalloc(3, int);
    arr[1] = stalloc(2, int);
    arr[0][0] = 1;
    arr[0][1] = 3;
    arr[0][2] = 5;
    arr[1][0] = 3;
    arr[1][1] = 9;
    return arr[0][0] + arr[1][arr[0][0]] + arr[arr[0][1]-arr[1][0]][arr[0][2]-arr[1][0]];
}
