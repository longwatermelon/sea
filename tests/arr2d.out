RETURN_CODE:15
STDOUT:
